rust/libnewsboat/src/utils.rs

```diff
@@ -200,5 +218,34 @@ mod tests {
             assert!(is_valid_color(color));
         }
     }
+
+    #[test]
+    fn t_is_valid_attribte() {
```

Comment:
Please port this test to C++, so we can be sure that FFI didn't break anything.

For functions without tests, it's best to write tests beforehand to make sure that they actually pass on the original C++ implementation. In this case, the function is pretty trivial, so don't bother with that; but please keep it in mind for the future.